specification/automation/resource-manager/Microsoft.Automation/2017-05-15-preview/definitions.json

```diff
@@ -59,38 +59,41 @@
                     "readOnly": true
                 }
             },
-            "required": [ "updateConfiguration" ]
+            "required": [
+                "updateConfiguration",
```

Comment:
I believe that @fearthecowboy / @olydis require that we use two-spaced formatting. Please change that back :)

Reply:
Will do...